### ISSUE TITLE
Is model works with actual data?

### ISSUE THREAD

**Issue by albert1480** [michaelba8/Bitcoin-prediction-ML-project#1]

**Comment by michaelba8** [comment]
right now it is not perfect because the data used for the ML is not updated.

**Comment by albert1480** [comment]
I see. So I need make script for live downloading data of btc and save as btc files?

**Comment by michaelba8** [comment]
yes, the most updated month in the data is October-2020 if you get the last 4 months, you will get more accurate results.
you can download and try the code yourself, the file bitcoin_prediction.py contains all the function and the file simulation.py is testing data of the last 25 hours.
 
the project is not final yet.

**Comment by albert1480** [comment]
Ok. Thank you for your advice
